fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrivilegeChecker.java

```diff
@@ -1047,14 +1052,33 @@ public Void visitShowRoutineLoadStatement(ShowRoutineLoadStmt statement, Connect
             return null;
         }
 
+        @Override
+        public Void visitShowRoutineLoadTaskStatement(ShowRoutineLoadTaskStmt statement, ConnectContext session) {
+            String dbName = statement.getDbFullName();
+            String jobName = statement.getJobName();
+            RoutineLoadManager routineLoadManager = GlobalStateMgr.getCurrentState().getRoutineLoadManager();
+            String tableName = "";
+            try {
+                RoutineLoadJob routineLoadJob = routineLoadManager.getJob(dbName, jobName);
```

Comment:
Why add this logic in the position of permission verification？

Reply:
Because need to get tablename through jobname，Determine whether there is load permission for the table

Reply:
For permission verification, I agree with your modification method. But I didn't see this logic in the old code, adding permission verification logic may lead to inconsistent user interface between different versions. If you need to add this feature, you can submit a separate PR instead of modifying the user-visible interface in the refactored code.

Reply:
The reference is ShowExecutor.handleShowRoutineLoadTask()  , Please help to check again ，thank you
@HangyuanLiu